src/data/model/LayerTreeNode.js

```diff
@@ -90,9 +98,67 @@ Ext.define('GeoExt.data.model.LayerTreeNode', {
         this.callParent(arguments);
 
         // forward changes to ol object
-        if (key === 'checked') {
+        if (!this.__bubbling && key === 'checked') {
+            if(this.get('isLayerGroup') && this.get('toggleMode') === 'classic'){
```

Comment:
What do you think: Should we alias `this` to `me`? It looks as if it is used a lot in this method…